### DISCUSSION TITLE
Laravel can’t kept auth after login

### DISCUSSION THREAD

**Discussion by esmaily** [laravel/framework#39496] [Q&A]
- Laravel Version: 6.18.11
- PHP Version: 7.4.22
- Database Driver & Version: mysql

### Description:

I am trying to test login in Laravel 6.18 please help me.
Login with auth()->loginUsingId() or Auth::attempt login but is not working can not keep login
### Steps To Reproduce:

Please help me

**Comment by esmaily** [comment]
I'm confused please help me @taylorotwell

**Comment by dennisprudlo** [comment]
Any code or config to show us? "Steps to reproduce": "Please help me" isn't much of a help 😅

**Comment by ITwrx** [comment]
some wild guesses:

your php sessions are borked? perms, php config? check your logs? 
laravel 6 isn't truly compat with php 7.4.22? laravel docs might say ">= php 7.2", but how could they predict the future? try with older php like 7.2.x?
